src/api/parser/src/utils/wiki-feed-parser.js

```diff
@@ -86,7 +86,7 @@ module.exports = async () => {
       // Make sure the URL for this feed is a valid http/https web URI,
       // then process into a Feed object.
       try {
-        new URL(currentFeedInfo.url);
```

Comment:
It's ok to put an eslint ignore rule on the line above, you need `new` to calling the constructor.

Reply:
Done. Thanks.